src/Symfony/Component/Lock/StoreInterface.php

```diff
@@ -11,26 +11,18 @@
 
 namespace Symfony\Component\Lock;
 
-use Symfony\Component\Lock\Exception\LockAcquiringException;
 use Symfony\Component\Lock\Exception\LockConflictedException;
-use Symfony\Component\Lock\Exception\LockReleasingException;
 use Symfony\Component\Lock\Exception\NotSupportedException;
 
 /**
  * StoreInterface defines an interface to manipulate a lock store.
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
+ *
+ * @deprecated "Symfony\Component\Lock\StoreInterface" is deprecated since Symfony 4.4 and has been split into "Symfony\Component\Lock\PersistStoreInterface", "Symfony\Component\Lock\BlockingStoreInterface".'
  */
-interface StoreInterface
+interface StoreInterface extends PersistStoreInterface
```

Comment:
Shouldn't `StoreInterface` implements `BlockingStoreInterface` as well (and methods from that interface removed here)?

Reply:
We have the support method in the BlockingStoreInterface that should not appear in the StoreInterface or it will break BC.

Reply:
ok, fair enough.